spy/tests/compiler/test_exception.py

```diff
@@ -15,6 +15,8 @@ def foo(x: i32) -> i32:
                 raise Exception("hello")   # <-- line 6
             elif x == 2:
                 raise ValueError("world")
+            elif x == 10:
+                raise ValueError()
```

Comment:
`mod.foo(3)` vs `mod.foo(10)` now is a bit confusing, it took a while to me to understand what was going on.
I suggest to use `x == 3` for `ValueError()`, and later call `mod.foo(4)` for the `IndexError` case